app/javascript/mastodon/features/ui/components/alt_text_warning_modal.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { injectIntl, FormattedMessage } from 'react-intl';
+import Button from '../../../components/button';
+import { closeModal } from '../../../actions/modal';
+
+const makeMapStateToProps = () => {
+  const mapStateToProps = state => ({
+    media: state.getIn(['compose', 'media_attachments']),
+  });
+
+  return mapStateToProps;
+};
```

Comment:
This is actually entirely pointless, I don't use the `media` prop at all, but I couldn't figure out how to get rid of it without this component breaking. Anyone know what I should replace this with?